demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -962,15 +962,15 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
       document.getElementById('audio-preview').parentElement.style.visibility = 'hidden';
```

Comment:
Do we need to update this to getByteTimeDomainData too? Also does this check work?

Reply:
Good call. Thanks.